apps/public-docsite-v9/src/Concepts/Theming.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Theming" />
+
+# Theming
+
+Fluent UI Theme is represented by a set of tokens. Each token resolves to a single value which can be assigned to a CSS property.
+
+You can browse all the available tokens in **Theme** section of the docs.
```

Comment:
You can actually hyperlink to other pages or other parts of the same page. You can find some examples on the docs of build time optimization of Griffel

Reply:
Yeah, I know. the problem is Theme is a non-linkable section, not a page :-/

Reply:
maybe just link to the most 'natural' part of that section like the color tokens ?